packages/components/src/components/page/page.scss

```
@@ -13,20 +22,18 @@
 	}
 
 	&[data-variant="fixed"] {
+
+		@include header-footer-style();
 		block-size: 100%;
 		min-block-size: 100%;
 		display: flex;
 		flex-direction: column;
-
-		header,
-		footer {
-			display: flex;
-			flex: 0 1 auto;
-			flex-grow: 0;
-			flex-shrink: 0;
+		/* workaround for angular */
+		> :is(db-header, dbheader, dbfooter, db-footer) {
```

```suggestion
		> :is(.db-header, .db-footer, db-header, dbheader, dbfooter, db-footer) {
```

why don't you just leave this in here, and add further selectors to the list?

---

The mixin `@include header-footer-style();` has those selectors in it already, this would be only a duplicate